ipatests/test_integration/test_otp.py

```diff
@@ -328,11 +335,6 @@ def setup_otp_nsslapd(self):
         tasks.ldapmodify_dm(self.master, ldap_query.format(limit=new_limit))
         # Be sure no services are running and failed units
         self.master.run_command(['killall', 'ipa-otpd'], raiseonerr=False)
```

Comment:
Is it needed to keep this second killall command? The test just did killall, modify timeout, killall.

Reply:
No its not required. I will remove that.